includes/derivatives.inc

```diff
@@ -486,3 +485,28 @@ function islandora_ocr_set_generate_ocr_rels_ext_statement(AbstractObject $objec
   $generate_ocr = $generate_ocr ? 'TRUE' : 'FALSE';
   $object->relationships->add(ISLANDORA_RELS_EXT_URI, 'generate_ocr', $generate_ocr, RELS_TYPE_PLAIN_LITERAL);
 }
+
+/**
+ * Remove the generate_hoct field if present in the RELS-EXT.
+ */
+function islandora_ocr_remove_generate_hocr_rels_ext_statement(AbstractObject $object) {
```

Comment:
Two copies of much the same functions now... should we parameterize? Possibly introduce constants for the flags? Deprecate the present functions, and make 'em call our parameterized version?